web/src/components/MemoEditor.tsx

```diff
@@ -77,13 +77,16 @@ const MemoEditor: React.FC = () => {
   }, [state, editorState.editMemoId]);
 
   const handleKeyDown = (event: React.KeyboardEvent) => {
-    if (event.key === "Escape") {
-      if (state.fullscreen) {
-        handleFullscreenBtnClick();
-      }
-    } else if (event.key === "Enter" && (event.ctrlKey || event.metaKey)) {
+    if (event.key === "Escape" && state.fullscreen) {
+      handleFullscreenBtnClick();
+    }
+    if (event.key === "Enter" && (event.ctrlKey || event.metaKey)) {
       handleSaveBtnClick();
     }
+    if (event.key === "Tab") {
```

Comment:
`if-else` is more understandable, wdyt?

```suggestion
    } else if (event.key === "Enter" && (event.ctrlKey || event.metaKey)) {
      handleSaveBtnClick();
    } else if (event.key === "Tab") {
```

Reply:
We need else if or return to avoid going into two branches